src/main/java/frc/robot/subsystems/RotatorSubsystem.java

```diff
@@ -59,7 +59,7 @@ public void reset() {
         setSetpoint(Math.toDegrees(getMeasurement()));
     }
 
-    private double getMeasurement() {
+    public double getMeasurement() {
```

Comment:
Don't expose this one. The public interface of this subsystem is in degrees. This one is in radians. Much sadness will follow.

Reply:
Ah, yes we found this sadness when testing yesterday. Fixed!